style/ViewDataset.mcss

```diff
@@ -60,22 +60,22 @@
   z-index: 1;
 }
 
-.ViewDataset_LeftSidebar > * {
+.leftSidebar > * {
   flex: none;
 }
 
-.ViewDataset__Modules {
+.modules {
   flex: 1;
   display: flex;
   flex-direction: column;
 }
 
-.ViewDataset__Modules > * {
+.modules > * {
```

Comment:
I'll look through them and do my best.  Some of them I couldn't since Link does not take a className parameter.